Makefile

```diff
@@ -77,3 +81,9 @@ swagger:
 
 download_test_data:
 	./test/data/download.sh genomic_small
+	./test/data/download.sh i2b2
```

Comment:
Actually the goal of adding the parameter was to not have to download everything all the time. Is the i2b2 data needed in the test? It doesn't look like it but I might have missed something.

Reply:
yes, we need them when testing the loader

Reply:
:( OK